c/tskit/tables.h

```diff
@@ -3539,10 +3547,13 @@ different error conditions. Please see the
 sequentially load tree sequences from a stream.
 
 Please note that this streaming behaviour is not supported if the
-TSK_LOAD_SKIP_TABLES option is set. With this option, only the top-level
-information of the table collection will be read, leaving all tables empty. When
-attempting to read from a stream with multiple table collection definitions and
-the TSK_LOAD_SKIP_TABLES option set, only the top-level information of the first
+TSK_LOAD_SKIP_TABLES or TSK_LOAD_SKIP_REFERENCE_SEQUENCE option is set.
+If the TSK_LOAD_SKIP_TABLES option is set, only the top-level
+information of the table collection will be read, leaving all tables empty.
```

Comment:
```suggestion
If the TSK_LOAD_SKIP_TABLES option is set, only the non-table
information from the table collection will be read, leaving all tables with zero rows and no metadata or schema.
```

Reply:
thanks @benjeffery! since that's the `skip_tables` docstring, should I open another PR for that? or are y'all ok with throwing that in with the `skip_refseq` PR?

Reply:
It should be in this PR as that text only needs changing because of the ref seq.